query-engine/core/src/query_document/transformers.rs

```diff
@@ -10,11 +10,11 @@ use chrono::prelude::*;
 use prisma_models::{OrderBy, PrismaValue, ScalarFieldRef};
 use std::convert::TryInto;
 
-impl TryInto<PrismaValue> for ParsedInputValue {
```

Comment:
Was this a clippy warning?

Reply:
No, but somewhere in the changes made in this PR, it was easier to use `TryFrom`. Since it is recommended to implement `TryFrom` instead of `TryInto` (since `TryFrom` gives you `TryInto` for free), I just took the time to convert all implementations. It should be 100% equivalent. Lemme know if that's an issue or if there was any reasons not to implement `TryFrom`